core/src/test/java/org/infinispan/distribution/ZeroCapacityNodeTest.java

```diff
@@ -0,0 +1,57 @@
+package org.infinispan.distribution;
+
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.distribution.ch.ConsistentHash;
+import org.infinispan.distribution.ch.impl.DefaultConsistentHash;
+import org.infinispan.distribution.group.impl.PartitionerConsistentHash;
+import org.infinispan.remoting.transport.Address;
+import org.infinispan.test.MultipleCacheManagersTest;
+import org.infinispan.test.TestingUtil;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+/**
+ * Test the capacity factor for lite instance
+ *
+ * @author Katia Aresti
+ * @since 9.4
+ */
+@Test(groups = "functional", testName = "distribution.ch.ZeroCapacityNodeTest")
+public class ZeroCapacityNodeTest extends MultipleCacheManagersTest {
+
+   public static final int NUM_SEGMENTS = 60;
+
+   @Override
+   protected void createCacheManagers() throws Throwable {
+      // Do nothing here, create the cache managers in the test
+   }
+
+   public void testCapacityFactorForLiteInstance() {
```

Comment:
I was expecting more tests to be honest. 
* Does both cache mode behaves correctly with zero and non-zero nodes? 
* Will counter and multimap work correctly? 
* what happens if you start a cache in a non-zero node? Will all the zero nodes start the cache as well?